presto-cache/src/test/java/com/facebook/presto/cache/alluxio/TestAlluxioCacheConfig.java

```diff
@@ -68,6 +73,9 @@ public void testExplicitPropertyMappings()
                 .put("cache.alluxio.quota-enabled", "true")
                 .put("cache.alluxio.shadow-cache-enabled", "true")
                 .put("cache.alluxio.shadow-cache-window", "1d")
+                .put("cache.alluxio.ttl-enabled", "true")
+                .put("cache.alluxio.ttl-check-interval", "60s")
```

Comment:
better to keep them alphabetical

```suggestion
                .put("cache.alluxio.ttl-check-interval", "60s")
                .put("cache.alluxio.ttl-enabled", "true")

```